### DISCUSSION TITLE
podman.socket is masked

### DISCUSSION THREAD

**Discussion by zakaria-chahboun** [containers/podman#17652] [Q&A] [answered]
I want to connect [pods](https://github.com/marhkb/pods) app with podman socket,

I did this:
```console
systemctl enable --now --user podman.socket
```
I got:
> Failed to enable unit: Unit file /etc/xdg/systemd/user/podman.socket is masked.

Then i did:
```console
sudo systemctl enable --now podman.socket 
```

I got:
> Job failed. See "journalctl -xe" for details.

```console
Feb 28 12:22:27 geo sudo[18504]: pam_unix(sudo:session): session closed for user root
Feb 28 12:22:36 geo sudo[18543]:     zaki : TTY=pts/1 ; PWD=/home/zaki ; USER=root ; COMMAND=/usr/bin/systemctl enable --now podman.socket
Feb 28 12:22:36 geo sudo[18543]: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1000)
Feb 28 12:22:36 geo systemd[1]: Reloading.
Feb 28 12:22:36 geo systemd[1]: podman.socket: Socket service podman.service not loaded, refusing.
Feb 28 12:22:36 geo systemd[1]: Failed to listen on Podman API Socket.
```

Any solution for this issue? Thanks :rose: 

I have `podman version 4.3.1`

**Comment by Luap99** [comment, marked as answer]
systemd units should not be masked by default so this likely something you did.
To unmask a unit use `systemctl unmask ...`

You may want to also check that your units match our upstream ones: https://github.com/containers/podman/tree/main/contrib/systemd/system

**Reply by zakaria-chahboun** [reply]
Thanks @Luap99 The issue was in the units, i Just replaced it with the https://github.com/containers/podman/tree/main/contrib/systemd/system and all things is good now!

**Reply by qdm12** [reply]
Hey @zakaria-chahboun I have the exact same problem after doing `sudo systemctl unmask podman.socket`. Would you have any other clue on how to fix this? Thanks!
